test/check-img.js

```diff
@@ -0,0 +1,22 @@
+import fs from 'fs';
+import path from 'path';
+
+import {assert} from 'chai';
+
+import {languages} from '../lib/languages';
+
+const img_dir = path.resolve('views/resources/logos');
+
+function checkImage(lang) {
+    let result = fs.existsSync(img_dir + '/' + lang.logoUrl);
```

Comment:
Should change the concat with a `path.join` call so this can run on Windows

Reply:
I did all my development and it worked fine